src/component/layout/Header/Header.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { TbCat } from "react-icons/tb";
+import { TbDog } from "react-icons/tb";
+import { TbSearch } from "react-icons/tb";
+// import { TbHeart } from "react-icons/tb";
+import "./Header.css";
+
+const Header = () => {
+  return (
+    <>
+      <div className="header">
+        <div className="header_wrap">
+          <div className="logo_wrapper">
+            <div className="logo">
+              <TbCat size="30px" />
+              <TbDog size="30px" />
+            </div>
+          </div>
+          <ul className="header_navBar">
+            <li className="navBar_item">Home</li>
+            <li className="navBar_item">Pets</li>
+            <li className="navBar_item">Donate</li>
+            <li className="navBar_item">Contact</li>
+            <li className="navBar_item">About</li>
+          </ul>
```

Comment:
I would suggest using <a> right away  since are going to use links here anyway. Also, you might want to create an array and loop over it. As a 'TODO' you could add a feature that shows which link is active for the future.

Reply:
Thank you Oksana. Good to know that we can put comments here. Never seen it before.